x-pack/plugin/sql/qa/server/src/main/resources/agg-nulls-zeros.csv-spec

```diff
@@ -0,0 +1,359 @@
+
+aggregatingAllZerosWithFirst
```

Comment:
No need to prefix all tests with `aggregating` - that already happens through the file name.